ci-operator/jobs/infra-periodics.yaml

```diff
@@ -59,6 +59,48 @@ periodics:
     - name: token
       secret:
         secretName: github-credentials-openshift-bot
+- agent: kubernetes
+  cluster: app.ci
+  cron: '@yearly'
+  decorate: true
+  labels:
+    ci.openshift.io/role: infra
+  name: periodic-rotate-serviceaccount-secrets
+  spec:
+    containers:
+    - args:
+      - --namespace=ci
+      - --dry-run=true
```

Comment:
I assume this will be flipped afterwards.

Reply:
Yeah I mostly left it so we don't get surprised once in a year :upside_down_face: